imagescan/controller.go

```diff
@@ -211,6 +212,9 @@ func (s *Controller) findPendingImages() []*image {
 }
 
 func (s *Controller) scanImages(ctx context.Context, images []*image) error {
+	if err := s.updateImageScanStatus(ctx, images, castai.ImageScanStatusPending); err != nil {
```

Comment:
Check https://github.com/castai/kvisor/blob/ba22c3740905633df9ecff5d0fbe987eb6dbd7e7/imagescan/controller.go#L167

Initially would be good to send all pending images. But we also don't want to send it everytime. Could have some global field initialPendingImagesSent similar to fullSnapshotSent